analysis.R

```diff
@@ -1,138 +1,129 @@
-###################
-##  12/27/21  ##
-###################
+#####################################
+##  Depression-Belief Interaction  ##
+#####################################
 
 ####  Startup  ####
-
 # Load packages
-
-if(!require(tidyverse)){install.packages('tidyverse')}
```

Comment:
Not necessary, since you only install packages once. This code can be helpful to someone who's new to R but I find it kind of busy